packages/perseus/src/util/sre/tex-to-text.ts

```diff
@@ -0,0 +1,37 @@
+import {liteAdaptor} from "mathjax-full/js/adaptors/liteAdaptor";
+import {STATE} from "mathjax-full/js/core/MathItem";
+import {SerializedMmlVisitor} from "mathjax-full/js/core/MmlTree/SerializedMmlVisitor";
+import {HTMLDocument} from "mathjax-full/js/handlers/html/HTMLDocument";
+import {TeX} from "mathjax-full/js/input/tex";
+import {AllPackages} from "mathjax-full/js/input/tex/AllPackages";
+
+import * as SRE from "./sre";
+
+import type {MmlNode} from "mathjax-full/js/core/MmlTree/MmlNode";
+
+// Code adapted from https://github.com/Speech-Rule-Engine/sre-latex/blob/main/tex2mml/tex2mml.js
+
+//  Busproofs requires an output jax, which we aren't using
+const packages = AllPackages.filter((name) => name !== "bussproofs");
+
+//  Create the input jax
+const tex = new TeX({packages: packages});
+
+//  Create an HTML document using a LiteDocument and the input jax
```

Comment:
It might be worth clarifying here that liteAdaptor doesn't touch the browser's actual DOM.